spec/timetrap_spec.rb

```diff
@@ -33,8 +33,22 @@ def with_stubbed_config options = {}
   end
 end
 
+module Timetrap::WithAttributes
+  def with_rounding_on
```

Comment:
🥦  Extracted helper for other cases where we might want to test attributes and have them reverted/reset after the test.